.github/workflows/linters.yml

```diff
@@ -0,0 +1,26 @@
+name: Linters, static checks and code coverage
+
+on:
```

Comment:
Could you add some documentation (in a README, in a comment, etc...) on what the next steps here are?

I have various open questions. E.g.:
1. When/who is responsible for updating the branches?
2. When/who is responsible for updating the paths?
3. When/who is responsible for adding linter checks?
4. Is there a repository of linter checks we reference or do we create our own?

No need to answer everything in line, but I'm just not sure what the next steps are.